### ISSUE TITLE
Save top-k predictions and scores for GOOG team 

### ISSUE THREAD

**Issue by hvgazula** [hassonlab/247-pickling#100]

**Comment by hvgazula** [comment]
Currently, I am saving all the logits. This could be a problem when we 'concatenate' logits from all conversations. I cannot test this unless I save the logits and check the file sizes.
